website/docs/container-only-builds.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Container-only builds
+---
+
+# Container-only builds
+
+It is possible to use Dalec when you wish to build a minimal image from scratch or based on one of Dalec's supported distros (see [Targets](targets.md) for a list of these) with only certain packages installed. To do this, simply define a Dalec spec with only runtime dependencies specified. The resulting image will contain only the specified packages and their dependencies.
+
+```yaml
+name: my-minimal-image
+version: 0.1.0
+description: A minimal distroless image with only curl and shell access
+revision: 1
+
+dependencies:
+    runtime:
+        curl:
+        bash:
+
+image:
+    entrypoint: /bin/bash
+
+```
+
+Then, to build:
+`docker build -f my-minimal-image.yml --target=mariner2 -t my-minimal-image:0.1.0 .`
```

Comment:
do we need a docker version requirement or use `buildx build` explicitly? in older version of docker, `build` doesn't symlink to `buildx build`. do we think we need to highlity?

Reply:
Oh that is a really good point. I will use `docker buildx build` in the example here and then call that out.